bookmarks/urls.py

```diff
@@ -32,5 +33,7 @@
     path('feeds/<str:feed_key>/all', AllBookmarksFeed(), name='feeds.all'),
     path('feeds/<str:feed_key>/unread', UnreadBookmarksFeed(), name='feeds.unread'),
     # Health check
-    path('health', views.health, name='health')
+    path('health', views.health, name='health'),
+    # Manifest
+    path("manifest.json", TemplateView.as_view(template_name="metadata/manifest.json", content_type="application/json"), )
```

Comment:
Unfortunately this is a bit more complex: linkding allows configuring a context path via `LD_CONTEXT_PATH`, which allows to host it under a URL like `https://mydomain.com/linkding`. In that case the context path needs to be respected for the manifest location and the scope as well. I think this needs the following changes:
- Define a name for the manifest route, similar to the other routes here
- Generate the effective URL for that route in `layout.html` using the `url` template tag, there are existing examples in that file
- Create a view function for the manifest, add the context path to `scope`. Instead of rendering that view from a template, I would suggest to create the JSON in the view function, see `health.py` for an example. Otherwise you'd have to figure out how to render a template into a JSON response.

Reply:
I'll get those changes made over the next couple of days.

Reply:
I have updated the way the manifest is generated. I have run some tests with and without a context path and everything appears to be working as expected